ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.15-nightly-x86.yaml

```diff
@@ -46,7 +46,7 @@ tests:
       OPENSHIFT_VERSION: "4.15"
```

Comment:
Again, do we want report profile for the 249node test?

Reply:
metrics-aggregated.yml or metrics.yml may timeout (and index large number of docs to ES) in 249 node environment, so better we can use report mode.

Reply:
Ack, updated - ptal

Reply:
I agree that we don't want report profile for 4.15. However metrics-aggregated.yml and metrics.yml getting timeoutd on large scale. So this is the need of the hour.
We will remove report profile on 4.15 once https://github.com/kube-burner/kube-burner-ocp/pull/3 merged and similar change available in metrics.yml